src/map/battle.cpp

```diff
@@ -3284,6 +3283,10 @@ static bool is_attack_hitting(struct Damage* wd, struct block_list *src, struct
 	}
 
 	hitrate = cap_value(hitrate, battle_config.min_hitrate, battle_config.max_hitrate);
+
+	if(skill_id == PA_SHIELDCHAIN)
```

Comment:
No, this is intentional, please see the comment.

If your min_hitrate is 5% then Shield Chain should have a minimum hit rate of 20.

Reply:
I thought this is a custom *Athena configuration and not official behavior?
If I set that maximum to 10 I wouldnt expect a skill to exceed the value of 10.

Reply:
But if all other skills are officially 5% minimum and Shield Chain is 25% minimum hitrate. If someone changes it to 10% (+5% from official value), I think they would expect Shield Chain to becomes 30% and not 10%. Do you think otherwise?